### PR TITLE
feat: Allow to gotestmd generate suites based on custom suite pkg

### PR DESCRIPTION
**Pull request by denis-tingaikin**
## Motivation

This extension point could simplify and help developers add different extensions. 
For example: 
1. Collecting logs from the test.
2. Do some preparation Before/After Test/Suite and so on
3. User alternative implemented running tool.

### CODE REVIEW

**Comment by denis-tingaikin** [comment]
Need to think about this direction, probably have a more clear way to collecting test logs

File: README.md [hunk 1]
```diff
@@ -4,10 +4,19 @@ Tool to generate go tests based on markdown files.
 
 ## Usages
 
+Generate suties with default bash runner:
```
Comment (haiodo) [inline README.md:7]:
```suggestion
Generate suites with default bash runner:
```

File: internal/config/config.go [hunk 2]
```diff
@@ -27,15 +27,23 @@ import (
...
-	if len(os.Args) != 3 {
-		logrus.Fatal("ARGs have wrong length. Expected: input-dir output-dir")
+	if len(os.Args) < 3 || len(os.Args) > 4 {
+		logrus.Fatal("ARGs have wrong length. Expected: (string)input-dir (string)output-dir (string)base-pkg[optional]")
 	}
-	return Config{
+	result := Config{
 		InputDir:  os.Args[1],
 		OutputDir: os.Args[2],
+		BasePkg:   "github.com/networkservicemesh/gotestmd/pkg/suites/shell",
```
Comment (haiodo) [inline internal/config/config.go:41]: Why this is default?

**Review by haiodo — APPROVED** [review]

File: internal/config/config.go (hunk 2 above)
Comment (denis-tingaikin) [reply internal/config/config.go:41]: Currently, this is using by default in the master implementation.

**Review by haiodo — APPROVED** [review]

**Pull request merged**